src/Constants.py

```diff
@@ -24,10 +26,10 @@
 
 # Network constants
 DEFAULT_NETWORK_CONFIG_MAP = {
-    'MAINNET': {'NAME': 'MAINNET', 'NB_FREEZE_CYCLE': 5, 'BLOCK_TIME_IN_SEC': 60, 'BLOCKS_PER_CYCLE': 4096,
-                'BLOCKS_PER_ROLL_SNAPSHOT': 256, 'BLOCK_REWARD': 40000000, 'ENDORSEMENT_REWARD': 1250000},
-    CURRENT_TESTNET: {'NAME': CURRENT_TESTNET, 'NB_FREEZE_CYCLE': 3, 'BLOCK_TIME_IN_SEC': 30, 'BLOCKS_PER_CYCLE': 2048,
-                      'BLOCKS_PER_ROLL_SNAPSHOT': 128, 'BLOCK_REWARD': 40000000, 'ENDORSEMENT_REWARD': 1250000},
+    'MAINNET': {'NAME': 'MAINNET', 'NB_FREEZE_CYCLE': 5, 'BLOCK_TIME_IN_SEC': 30, 'BLOCKS_PER_CYCLE': 8192,
+                'BLOCKS_PER_ROLL_SNAPSHOT': 512, 'BLOCK_REWARD': 20000000, 'ENDORSEMENT_REWARD': 78125},
+    CURRENT_TESTNET: {'NAME': CURRENT_TESTNET, 'NB_FREEZE_CYCLE': 3, 'BLOCK_TIME_IN_SEC': 15, 'BLOCKS_PER_CYCLE': 4096,
+                      'BLOCKS_PER_ROLL_SNAPSHOT': 256, 'BLOCK_REWARD': 20000000, 'ENDORSEMENT_REWARD': 78125},
```

Comment:
where did you find the numbers for the test network?

Reply:
Here: https://granadanet.tzkt.io/PtGRANADsDU8R9daYKAgWnQYAJ64omN1o3KMGVCykShA97vQbvV/constants